layers/meta-balena-jetson/recipes-kernel/linux/linux-tegra_%.bbappend

```diff
@@ -75,6 +76,10 @@ SRC_URI:append:cnx100-xavier-nx = " \
     file://tegra194-xavier-nx-cnx100.dtb \
 "
 
+SRC_URI:append:jetson-tx2-nx-devkit = " \
+    file://tegra186-p3636-0001-p3509-0000-a01-auvidea-jn30d.dtb \
+"
+
```

Comment:
@acostach  We did not have this section at all. Can you confirm if the `jetson-tx2-nx-devkit` device is the correct device for a emmc based tx2?

Reply:
@compiaffe Yes, the jetson-tx2-nx-devkit is for the eMMC Jetson TX2 NX module in the Xavier NX Devkit (or some other carrier board depending on the custom dtb like for this case)